src/components/Page/Page.stories.module.css

```diff
@@ -0,0 +1,13 @@
+.showcase-container {
+  background-color: rgb(168, 168, 168);
+  width: 100vw;
+  height: 100%;
+}
+
+.showcase-container__page-container {
+  padding-top: 10rem;
+  margin-top: 10rem;
+  margin: auto;
+  width: 60vw;
+  height: 60vh;
+}
```

Comment:
Are these styles really nescessary? As mentioned earlier (in another PR), the stories should ideally contain no other styles, as it may give a false impression of how the component actually looks.

Reply:
Only thing is that PageContent is white like it is supposed to be, and the storybook background is white so PageContent won't be visible. Maybe I should add shadow?

Reply:
Storybook also supports displaying a grid background, and changing the color. Setting a fixed background color on the component's story would IMO give a false impresssion of how the component looks in the end, and it will "break" Storybook functionality.
![image](https://user-images.githubusercontent.com/2082481/190342764-cad4a9b9-b95c-4ac5-a751-016b84656d6e.png)


Perhaps change the default background color for this story instead if this is important to demonstrate? Docs on background for Storybook: https://storybook.js.org/docs/react/essentials/backgrounds#gatsby-focus-wrapper

Reply:
I think this gives the wrong impression of the component
<img width="488" alt="image" src="https://user-images.githubusercontent.com/31471142/190350244-f51820b5-8823-45c3-984c-54a5c4f6b0ca.png">

Reply:
But perhaps a bit unnecessary to config it now as no one else than me cares about it, so I'll wait until it's necessary.

Reply:
How do you mean? This is how it will look if you add this component to a page with a white background.

Reply:
Yes

Reply:
So what do you think is the wrong impression about it? 🙂

Reply:
That at first galnce may seem like PageContent has transparent background color.

Reply:
That is a valid point, and that will likely be the case for other components as well.

I still dont think its the right thing to make changes to the story preview of this component, but perhaps we should consider changing the default background color for all stories to make it more clear that some components have a fixed background color. WDYT?

Reply:
Yes I think changing the background color for all stories would be wise. Perhaps to a neutral color that is not often used in our components.

Reply:
Great, then I think we agree 😁 So can we remove the extra styling here, and create a new PR to default to a neutral background (other than white)? Do you mind fixing this? 😇

Reply:
Sure :)